scripts/appveyor-tool.ps1

```diff
@@ -129,6 +129,7 @@ Function InstallRtools {
     $gcc_path = $env:GCC_PATH
   }
   $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\' + $gcc_path + '\bin;' + $env:PATH
+  $env:BINPREF=$RtoolsDrive + '/Rtools/mingw_$(WIN)/bin/'
```

Comment:
If you want to be able to compile packages with 32 and 64 bit at the same
time then yes.
On Mar 26, 2016 3:54 PM, "Kirill Müller" <notifications@github.com> wrote:

> In scripts/appveyor-tool.ps1
> <https://github.com/krlmlr/r-appveyor/pull/60#discussion_r57518650>:
>
> > @@ -129,6 +129,7 @@ Function InstallRtools {
> >      $gcc_path = $env:GCC_PATH
> >    }
> >    $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\' + $gcc_path + '\bin;' + $env:PATH
> > +  $env:BINPREF=$RtoolsDrive + '/Rtools/mingw_$(WIN)/bin/'
>
> Is this really necessary if PATH contains only the path to the appropriate
> bin directory (as selected by $gcc_path in the line above)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/krlmlr/r-appveyor/pull/60/files/d452a14e43fe0ae934bccb033d965cc46bf69bad#r57518650>
>